proposals/lip-0057.md

```diff
@@ -1903,7 +1903,7 @@ The following steps are executed as part of the (non-genesis) block processing,
 
 #### After Transactions Execution
 
-After the transactions in a block `b` are executed, the properties related to missed blocks are updated according to [Delegate Productivity][lip-0023#delegate-productivity]. This logic is recapitulated below:
+After the transactions in a block `b` are executed, the properties related to missed blocks are updated according to [Validator Productivity][lip-0023#delegate-productivity]. This logic is recapitulated below:
```

Comment:
We have naming mismatch when we referenced to the old LIPs. For a reader, it might looks odd if he is not aware of it.

Reply:
indeed, but I think that we agreed not to update old LIPs. Perhaps once we write the blogpost to inform the community for the change, we might add a sentence in the abstract or somewhere else that in LIP 57 we use the new terminology ans add link to the blogpost.